### PR TITLE
Add specification

### PR DESCRIPTION
**Pull request by SHtress**
Add .scs file with specification for component manager.
Need to update:
- Purpose
- Explanation
- Authors

### CODE REVIEW

File: specification.scs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    => nrel_sc_identifier: [SC-web specification](* <- lang_en;; *);
+    => nrel_key_sc_element: concept_animal;
+    => nrel_purpose: [SC-web specification purpose](* <- lang_en;; *);
+    => nrel_explanation: [SC-web explanation] (*<- lang_en;; *);
+    => nrel_authors: ... (* <- empty_set *);
+    => nrel_component_dependencies: ... (* <- empty_set;; *);;
+
+concept_dog
+    => nrel_install_scripts: [cd scripts && ./install_deps_ubuntu.sh](* <- lang_en;; *);
+    => nrel_install_scripts: [cd scripts && ./install.sh](* <- lang_en;; *);
```
Comment (SHtress) [inline specification.scs:22]: I think there shouldn't be English language. Probably there should be script language?

File: specification.scs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+sc_web_specification = [*
+sc_web
+    <- concept_reusable_component;
+    <- concept_atomic_reusable_component;
+    <- concept_independent_reusable_component;
+    <- concept_reusable_interface_component;
+    <- concept_reusable_dynamically_installed_component;
+    <- concept_reusable_source_code_component;
+
+    => nrel_sc_identifier: [SC-web specification](* <- lang_en;; *);
```
Comment (MksmOrlov) [inline specification.scs:13]: It is not SC-web specification. It is SC-web itself

File: specification.scs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    => nrel_key_sc_element: concept_animal;
```
Comment (MksmOrlov) [inline specification.scs:14]: hmm I am not sure

File: specification.scs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    => nrel_purpose: [SC-web specification purpose](* <- lang_en;; *);
+    => nrel_explanation: [SC-web explanation] (*<- lang_en;; *);
+    => nrel_authors: ... (* <- empty_set *);
+    => nrel_component_dependencies: ... (* <- empty_set;; *);;
+
+    => nrel_install_scripts: [cd scripts && ./install_deps_ubuntu.sh](* <- lang_en;; *);
+    => nrel_install_scripts: [cd scripts && ./install.sh](* <- lang_en;; *);
```
Comment (MksmOrlov) [inline specification.scs:21]: Make only one script

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: specification.scs [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    <- concept_reusable_source_code_component;
+
+    => nrel_sc_identifier: [SC-web specification](* <- lang_en;; *);
+    => nrel_purpose: [SC-web purpose](* <- lang_en;; *);
+    => nrel_explanation: [SC-Web is an intelligent knowledge base user interface] (*<- lang_en;; *);
+    => nrel_authors: ... (* <- empty_set *);
+    => nrel_component_dependencies: ... (* <- empty_set;; *);;
+
+    => nrel_install_scripts: [cd scripts && ./install_deps_ubuntu.sh](* <- lang_en;; *);
+    => nrel_install_scripts: [cd scripts && ./install.sh](* <- lang_en;; *);
```
Comment (smousy) [inline specification.scs:20]: It is changed for nrel_installation_script

**Review by smousy — CHANGES_REQUESTED** [review]

File: specification.scs [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    <- concept_independent_reusable_component;
+    <- concept_reusable_interface_component;
+    <- concept_reusable_dynamically_installed_component;
+    <- concept_reusable_source_code_component;
+
+    => nrel_sc_identifier: [SC-web specification](* <- lang_en;; *);
+    => nrel_key_sc_element: concept_animal;
+    => nrel_purpose: [SC-web specification purpose](* <- lang_en;; *);
+    => nrel_explanation: [SC-web explanation] (*<- lang_en;; *);
+    => nrel_authors: ... (* <- empty_set *);
```
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply specification.scs:16]: You can find some information for the specification from @NikitaZotov soft platform monography section

**Review by smousy — APPROVED** [review]

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
Check unresolved

File: specification.scs (hunk 6 above)
[the remark this answers is not in the document]
Comment (SHtress) [reply specification.scs:16]: Yes, there is some information but it is useless for component manager right now.

File: specification.scs (hunk 4 above)
Comment (SHtress) [reply specification.scs]: Well, if you insist.

File: specification.scs [hunk 7]
```diff
@@ -10,14 +10,12 @@ sc_web
...
-    => nrel_sc_identifier: [SC-web specification](* <- lang_en;; *);
-    => nrel_purpose: [SC-web purpose](* <- lang_en;; *);
+    => nrel_sc_identifier: [SC-web](* <- lang_en;; *);
     => nrel_explanation: [SC-Web is an intelligent knowledge base user interface] (*<- lang_en;; *);
     => nrel_authors: ... (* <- empty_set *);
     => nrel_component_dependencies: ... (* <- empty_set;; *);;
 
-    => nrel_installation_script: [cd scripts && ./install_deps_ubuntu.sh](* <- lang_en;; *);
-    => nrel_installation_script: [cd scripts && ./install.sh](* <- lang_en;; *);
+    => nrel_installation_script: [cd scripts && ./install_deps_ubuntu.sh && ./install.sh](* <- lang_en;; *);
```
Comment (MksmOrlov) [inline specification.scs:18]: remove lang_en

File: specification.scs (hunk 6 above)
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply specification.scs:15]: remove authors relation or specify it as not empty

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Review by MksmOrlov — APPROVED** [review]

**Pull request merged**